Makefile

```
@@ -145,21 +145,28 @@ generate: controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
 
 # Path used to import Gatekeeper manifests
-IMPORT_MANIFESTS_PATH ?= https://github.com/open-policy-agent/gatekeeper
+IMPORT_MANIFESTS_PATH ?= https://github.com/stolostron/gatekeeper
 TMP_IMPORT_MANIFESTS_PATH := $(shell mktemp -d)
 GATEKEEPER_MANIFEST_DIR ?= config/gatekeeper
 
+GATEKEEPER_TAG = $(shell curl -L https://api.github.com/repos/stolostron/gatekeeper/tags | jq -r '.[].name' | sort --version-sort | grep $(shell echo $(GATEKEEPER_VERSION) | cut -d '.' -f 1-3) | tail -1 )
+
 .PHONY: import-manifests
 import-manifests: kustomize ## Import Gatekeeper manifests.
+	if [ -z "$(GATEKEEPER_TAG)" ]; then \
+		echo "The gatekeeper version $(GATEKEEPER_VERSION) is not on the list" ;\
+		exit 1 ;\
+	fi 
```

Could we add a check that the GATEKEEPER_VERSION here matches the latest tag (in case we upgrade and forget to update it here):
```suggestion
	fi 
	if [ "$(GATEKEEPER_VERSION)" != "$${GATEKEEPER_TAG%-*}" ]; then \
	  echo "error: Gatekeeper tag $(GATEKEEPER_TAG) doesn't align with Gatekeeper version $(GATEKEEPER_VERSION) set in the operator.";
	  exit 1; \
	fi
```